modules/social_features/social_event/social_event.module

```diff
@@ -605,15 +672,16 @@ function social_event_show_pending_requests_message() {
         $query->condition($condition_group);
         $pending_requests = $query->execute();
         if (!empty($pending_requests)) {
-          $plural = \Drupal::translation()->formatPlural(count($pending_requests), 'is (1) new request', 'are (@count) new requests');
```

Comment:
@dmytriikaiun Looks like those changes not relevant and were added from code formatting of your IDE.